packages/runtime-core/src/component.ts

```diff
@@ -502,7 +502,7 @@ export function createComponentInstance(
     subTree: null!, // will be set synchronously right after creation
     effect: null!,
     update: null!, // will be set synchronously right after creation
-    scope: new EffectScope(true /* detached */),
+    scope: new EffectScope(false, parent?.scope),
```

Comment:
Why change this? I remember it was set to false to improve the performance.

Reply:
When I invoke the `pause` method of the `EffectScope` in a component instance, I hope to pause all the `effects` in the child components. However, the `EffectScope` in the child components and the `EffectScope` in the parent component are in a detached state, causing the pause effect to apply only to the current component. Therefore, I establish an association here.